CHANGELOG

```diff
@@ -59,9 +59,6 @@
 - fix #399.  Make sure {envtmpdir} is created if it doesn't exist at the
   start of a testenvironment run. Thanks Manuel Jacob.
 
-- fix #316: Lack of commands key in ini file is now treated as an error.
-  Reported virtualenv status is 'nothing to do' instead of 'commands
-  succeeded', with relevant error message displayed. Thanks Lukasz Rogalski.
 
```

Comment:
The changelog must not be changed retroactively, so this should not be removed, but instead en entry needs to be added, that this fix did not what was intended to do (throw a warning instead of causing an error) in the changelog for the current version.